pcsx2/Pcsx2Config.cpp

```diff
@@ -210,8 +210,8 @@ Pcsx2Config::GSOptions::GSOptions()
 	FramesToSkip			= 2;
 
 	LimitScalar				= 1.0;
-	FramerateNTSC			= 59.94;
-	FrameratePAL			= 50.0;
+	FramerateNTSC			= 59.29;
```

Comment:
I have never seen this number mentioned in a PSX doc (or in my own experiments).